Create map

MaciejKwik opened the pull request.

mslaws committed:
Create map

1aam2am1 commented:
Przejrzałem projekt mapy. O ile mapa jest prosta. Nie wiem dlaczego użyłeś terrain mesh, które o ile mi wiadomo jest technologią 3d. Z tego powodu **nieda się** wygenerować poligonu który mógłby służyć do kolizji z innymi obiektami. 
Obiekt Mesh Collider z pakietu Phisics, służy do ustawiania kolizji, niestety nie działa on z obiektami 2d. 
Proponuję użyć wbudowanej opcji Tilemap w unity w celu zbudowania ponownie mapy w technologii 2d. 
Inną opcją istnieje ręczne ustawianie wszystkich poligonów kolizji. Co nie jest opcją sugerowaną z powodu możliwych błędów, niedociągnięć oraz natłoku pracy.

Z powodu wyżej wymienionych uwag nie mogę na razie zaakceptować pull requesta.

maciekpajak committed:
New branch
-------------------------
+ StartScreen scene added
+ Menu scene added
+ PauseMenu in PlayScene added

Add files via upload

Add files via upload

Menu display bug fixed

Logowanie i Rejerstracja

New home and pause icons. Pause menu prefab created

Signed-off-by: Maciek Pająk <mpajak98@gmail.com>

Karolina-Lach committed:
armata, strzały, platforma, most

Do poprawy  ruchoma platforma i aktywowanie mostu

Karolina-Lach committed:
Moving platform fix

Fixed moving platform
TODO: bridge fix, explosion fix

Karolina-Lach committed:
Snail enemy

Added snail enemy

Karolina-Lach committed:
Box destroy

1aam2am1 committed:
Added Portal to move between scenes.

1aam2am1 committed:
Added Eagle and Bee Atack Modes.

1aam2am1 committed:
Added Background, Cv, Ground Coliders.

Karolina-Lach committed:
modify health

dodanie modify health do kuli z armaty, strzał i ślimaka

maciekpajak committed:
New items added

Signed-off-by: Maciek Pająk <mpajak98@gmail.com>

1aam2am1 committed:
Added shooting.

1aam2am1 committed:
Fast fix. Player can't change inventory in Edit Mode.

maciekpajak committed:
New items and new inventory graphics

Signed-off-by: Maciek Pająk <mpajak98@gmail.com>
